index-import.cpp

```diff
@@ -242,7 +225,30 @@ static bool cloneRecords(StringRef recordsDirectory,
 int main(int argc, char **argv) {
   cl::ParseCommandLineOptions(argc, argv);
 
-  auto remapper = Remapper::createFromCommandLine(PathRemaps);
+  Remapper remapper;
+  // Parse the the path remapping command line flags. This converts strings of
+  // "X=Y" into a (regex, string) pair. Another way of looking at it: each
+  // remap is equivalent to the s/pattern/replacement/ operator.
+  auto errors = 0;
+  for (const auto &remap : PathRemaps) {
+    auto divider = remap.find('=');
+    auto pattern = remap.substr(0, divider);
+    std::regex re;
+    try {
+      re = std::regex(pattern);
+    } catch (const std::regex_error &e) {
+      errs() << "Error parsing regular expression: '" << pattern << "':\n"
+             << e.what() << "\n";
+      errors++;
+    }
+    auto replacement = remap.substr(divider + 1);
+    remapper.addRemap(re, replacement);
+  }
+
+  if (errors) {
+    errs() << "Aborting due to errors.\n";
```

Comment:
Throughout the file these are `llvm::errs` and `llvm::outs`. I'm fine with either, but if we drop the `llvm::` scoping, would you mind updating the others to match?

Reply:
Since we're importing the `llvm` namespace, I went with the plain forms.